### PR TITLE
frontend/feat: #518 Integrating Facebook SDK

### PR DESCRIPTION
**Pull request by PraveenGongada**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description
Added Facebook SDK for tracking app events.

## Customer
`ny_user_onboarded` -> New user onBoarded
`ny_user_first_ride_completed` -> User has taken first ride
`ny_user_ride_completed` -> User ride completed

## Driver
`ny_driver_verify_otp` -> Driver verified login/signup OTP

### CODE REVIEW

**Comment by witcher-shailesh** [comment]
No label added

File: Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java [hunk 1]
```diff
@@ -1412,6 +1414,12 @@ public void firebaseLogEvent(String event) {
         mFirebaseAnalytics.logEvent(event, params);
     }
 
+    @JavascriptInterface
+    public void metaLogEvent(String event){
+        AppEventsLogger logger = AppEventsLogger.newLogger(context);
```
Comment (witcher-shailesh) [inline Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java:1419]: Can you add try/catch block here?

Comment (PraveenGongada) [reply Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java]: Done

File: Frontend/android-native/app/src/main/java/in/juspay/mobility/MainActivity.java [hunk 2]
```diff
@@ -143,6 +143,9 @@
 //import com.facebook.LoggingBehavior;
 //import com.facebook.applinks.AppLinkData;
 
+import com.facebook.FacebookSdk;
```
Comment (ks-shree) [inline Frontend/android-native/app/src/main/java/in/juspay/mobility/MainActivity.java:146]: @PraveenGongada Can you remove imports if it is not used

Comment (PraveenGongada) [reply Frontend/android-native/app/src/main/java/in/juspay/mobility/MainActivity.java:146]: This import will be used internally

**Pull request merged**